pennylane/circuit_drawer/circuit_drawer.py

```diff
@@ -65,6 +67,11 @@ def __init__(
         self.charset = charset
         self.show_variable_names = show_variable_names
 
+        if show_all_wires:
+            # if the provided wires include empty wires, make sure they are included
+            # as active wires
+            self.active_wires = wires.all_wires([wires, self.active_wires])
```

Comment:
Is there a reason why you don't just call the `extract_active_wires` function if `show_all_wires` is false?

Reply:
Oh, it's called above! Unless I am misinterpreting your comment

Reply:
Hmm, the name `active_wires` does not make a lot of sense here any more...But we could leave this to another refactor?

Reply:
Yep, agree.